docs/reference/builder.md

```diff
@@ -1470,6 +1474,116 @@ The `STOPSIGNAL` instruction sets the system call signal that will be sent to th
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
 
+## SHELL
+
+	SHELL executable parameters
+    
+The `SHELL` instruction allows the default shell used for the *shell* form of
+commands to be overridden. The default shell on Linux is `["/bin/sh", "-c"]`, and on
+Windows is `["cmd","/S","/C"]`. The `SHELL` instruction *must* be written in JSON
```

Comment:
It's a really minor nit, but IMO the Windows example here ought to have consistent spacing with the Linux example (`["cmd", "/S", "/C"]`).